features/account/customer_account/address_book/deleting_address_created_after_checkout.feature

```diff
@@ -25,3 +25,18 @@ Feature: Removing an address from my book
         And I delete the "Mike Ross" address
         Then I should be notified that the address has been successfully deleted
         And I should have a single address in my address book
+
+    @api
+    Scenario: Deleting address created after placing an order
```

Comment:
Why did you add another scenario?

Reply:
Mostly because step "And my billing address is fulfilled automatically through default address", as in API there is no such a think like automatically, but the step implementation we have now is a bit wrong. I think i will make refactor of Context ( or transformer) so it will be nice but in another PR.